.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
 install:
   - npm install
   - bundle install
-  - (cd Example && bundle exec pod install)
+  - (cd Example && bundle exec pod repo update && bundle exec pod install)
```

Comment:
Just a note, it's not important that `bundle exec repo update` happens inside `Example`

Reply:
Thanks, I'll change it

Reply:
Done